src/plugins/minimal_scene/MinimalScene.hh

```diff
@@ -173,10 +183,11 @@ namespace plugins
     /// Values is constantly constantly cycled/swapped/changed
```

Comment:
We need to use `void*` as the texture ID may either be a `GLuint` for OpenGL or an Objective-C identifier (`id<MetalTexture>`).

Documentation needs to be updated.